types/eslint-ast-utils/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictFunctionTypes": true,
+        "strictNullChecks": true,
+        "types": [],
+        "noEmit": true,
+        "forceConsistentCasingInFileNames": true,
+        "esModuleInterop": true,
```

Comment:
`esModuleInterop` should not appear on DT and should be removed. Your tests should write:

```ts
import astUtils = require("eslint-ast-utils");
```

To fix that failure you saw.

Also, trailing commas aren't allowed as this file must be valid JSON for the infra to be able to read it (whereas TS is less strict).

Reply:
I figured, I just wasn't sure how else to resolve it. Thanks!